project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -62,4 +83,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Event in database
     response.getWriter().println(CommonUtils.convertToJson(SpannerTasks.getEventById(event.getId()).get().toBuilder().build()));
   }
+
+/*
+ * @param text: String of text that includes event name and description
+ * @return categoryNames: returns selected names of labels that NLP API suggests for text
+*/
+  public ArrayList<String> getNlp(String text) throws IOException {
```

Comment:
Might just be personal preference, but I would put these NLP-related methods into its own class so that this servlet is at a higher level of just doing some processing and writing out an event instead of revealing how the NLP is done and handled

Reply:
+1

Reply:
done